src/Workspaces/CoreTestUtilities/ExportProviderCache.cs

```diff
@@ -88,6 +88,11 @@ public static ComposableCatalog CreateAssemblyCatalog(IEnumerable<Assembly> asse
                 }
             }
 
+            return CreateAssemblyCatalogNoCache(assemblies, resolver);
```

Comment:
You can remove the null checks now in CreateAssemblyCatalog, since they're always non-null since this method is always running after the static field initialization ran.

(I do like the cleanup though, thanks.)

Reply:
Also random thought/naming nit: should CreateAssemblyCatalogNoCache just be called "CreateAssemblyCatalog" and the caching one called GetOrCreate? I think that pattern is seen elsewhere and clarifies that "Create" doesn't always create something new.

Reply:
➡️ Modified to use `Create` and `GetOrCreate` consistently aside from the remaining issue in #25863